platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java

```diff
@@ -291,14 +297,21 @@ public SecurityManager getSecurityManager() {
     }
 
     public static <T> T runAsAdmin(PrivilegedAction<T> action) {
-        Set<Principal> principals = new HashSet<>();
-        principals.add(new RolePrincipal("admin"));
-        javax.security.auth.Subject subject = new javax.security.auth.Subject(true,
-                principals,
-                new HashSet(),
-                new HashSet());
+        return javax.security.auth.Subject.doAs(JAVA_ADMIN_SUBJECT, action);
+    }
 
-        return javax.security.auth.Subject.doAs(subject, action);
+    public static <T> T runAsAdminPea(PrivilegedExceptionAction<T> action)
```

Comment:
I agree it's a weird name too! The reason for a different name is if we overload the runAsAdmin then we have to cast all our lambda expressions that call this method because they will not know which one to use. So to avoid adding casts all over our codebase I went with another name. I originally was going to go with something like  runAsAdminWithException except i thought it sounded weird to have 'exception' in the method name. It sounds like that is preferred over the current one though so i will change it.